.github/workflows/deploy-gh-pages.yml

```diff
@@ -0,0 +1,27 @@
+on: [push]
+
+jobs:
+  check-clean-style:
```

Comment:
(Nitpick) this name is copied over from another workflow and we should change it to something more descriptive eg:

```suggestion
  deploy:
```

Reply:
Indeed you are right! Resolved!